ChangeLog

```diff
@@ -13,6 +13,7 @@ Release date: TBA
 * Improved signature of the ``__init__`` and ``__postinit__`` methods of the following nodes:
   - ``nodes.Delete``
   - ``nodes.If``
+  - ``nodes.AssignName``
```

Comment:
```suggestion
  - ``nodes.AssignName``
  - ``nodes.Delete``
  - ``nodes.If``
```

Reply:
This was intentional, if all the addition are at the top we're going to get a conflict for each MR.